bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -56,11 +58,11 @@
 
     // Note -- synchronized though I guess this should always be called from
     // gui thread...
-    public static synchronized String formatBrightness(Magnitude mag) {
+    private static synchronized String formatBrightness(Magnitude mag) {
```

Comment:
The `TelescopePosTableWidget` is exclusively a GUI concept, built and updated by the UI in the swing thread.  This method is synchronized because the shared `static` `DecimalFormat` is not thread safe.  It shouldn't matter if it is only used by the event loop thread but, why not synchronize it anyway I suppose.